README.md

```diff
@@ -8,5 +8,5 @@ For interactionless use, set `ALLOWLISTED_PLUGINS=example-plugin1,example-plugin
 
 | Plugin       | Description     |
 |--------------|-----------|
-| Twitter      | AutoGPT is capable of retrieving Twitter posts and other related content by accessing the Twitter platform.|
+| [desojo/autogpt-twitter](https://github.com/desojo/autogpt-twitter)      | Gives Auto-GPT the ability to interact with twitter users via the v1.1 API using Tweepy|
```

Comment:
Why do you link the original fork if new code will be maintained here?

Reply:
Maybe it would make sense to link the original fork in the third party list and the official version up here?

Reply:
I’d say don’t do this, the goal of including twitter here was to make it a central source of code for twitter plugins. Linking externally doesn’t help that